src/MarketingPageAcceptanceTests.Steps/Utils/Settings.cs

```diff
@@ -0,0 +1,44 @@
+﻿using Microsoft.Extensions.Configuration;
+
+namespace MarketingPageAcceptanceTests.Steps.Utils
+{
+    public class Settings
+    {
+        public string HubUrl { get; set; }
+
+        public string MarketingPageUrl { get; set; }
+
+        public string Browser { get; set; }
+
+        public DatabaseSettings DatabaseSettings { get; set; }
+
+        // workaround for not including UITest everywhere
+        public bool CreateNewSolution { get; set; }
+
+
+        public Settings(IConfiguration config)
+        {
+            HubUrl = config.GetValue<string>("hubUrl");
+            var envUrl = config.GetValue<string>("pbUrl");
+            MarketingPageUrl = $"{envUrl}/marketing/supplier/solution";
```

Comment:
Do the ac tests need to have a notion of our routing? 

What I mean - would it perhaps be better if instead of providing the url of PB and then figuring out MP's url, we'd just provide MP's url in the settings? 
```
            MarketingPageUrl = config.GetValue<string>("mpUrl");
```

This would obviously mean to also change the value that's in appsettings.json

Reply:
The tests need to route directly to the marketing pages, and we have no notion of that in the front end, hence why I've added that minor change